server/poll.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 type Poll struct {
+	CreatedAt         int64
 	Creator           string
 	DataSchemaVersion string
```

Comment:
Should we update DataSchemaVersion?

Reply:
I think we should. Units v1.0.0 we can do every change we like. After v1.0.0 I will care about backwards compatibility. See https://semver.org/#spec-item-4